python/cugraph/cugraph/datasets/dataset.py

```diff
@@ -148,8 +150,29 @@ def __download_csv(self, url):
             )
         return self._path
 
-    def unload(self):
+    def __download_dask_csv(self, url):
+        """
+        Downloads the .csv file from url to the current download path
+        (self._dl_path) via multiple GPUs, updates self._path with the
+        full path to the downloaded file, and returns the latest value
+        of self._path.
+        """
+        self._dl_path.path.mkdir(parents=True, exist_ok=True)
+
+        filename = self.metadata["name"] + self.metadata["file_type"]
+        if self._dl_path.path.is_dir():
+            blocksize = dcg.get_chunksize(url)
+            ddf = dask_cudf.read_csv(path=url, blocksize=blocksize)
+            self._path = self._dl_path.path / filename
+            ddf.to_csv(self._path, index=False)
 
+        else:
+            raise RuntimeError(
+                f"The directory {self._dl_path.path.absolute()}" "does not exist"
+            )
+        return self._path
```

Comment:
Since the purpose of this method is to download a file from a URL and write a CSV file, do we need dask for this or can we just use the existing `__download_csv()` method?  

I guess if the file is large enough to require multiple GPUs then we might need this, however, I'm wondering if it would be better to remove this and refactor `__download_csv()` so that it simply downloads the CSV from `url` using `urllib.request.urlretrieve()`, then use it for both cases.  
```python
   urllib.request.urlretrieve(url, self._path)
```